src/test/java/io/zold/api/RtTransactionTest.java

```diff
@@ -46,9 +46,41 @@ public void shouldObeyEqualsHashcodeContract() {
             .verify();
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void idIsNotYetImplemented() {
-        new RtTransaction("id()").id();
+    @Test
+    public void returnsId() throws IOException {
+        MatcherAssert.assertThat(
+            new RtTransaction(
+                "abcd;2017-07-19T21:25:07Z;0000000000a72366;xksQuJa9;98bb82c81735c4ee;For food;QCuLuVr4..."
+            ).id(),
+            // @checkstyle MagicNumber (1 line)
+            new IsEqual<>(43981L)
+        );
+    }
+
+    @Test(expected = IOException.class)
```

Comment:
@carlosmiranda In all those tests expecting exception, it would be best to validate the exception thrown are actually does we are expecting and not something else coming from another bug (such as wrong position in the transaction string).
`ExpectedException` rule is good for this.